docs/source/workflows/predictors.rst

```diff
@@ -838,3 +838,63 @@ Because training data are shared by all predictors in the graph, a data source d
+
+Once a :class:`~citrine.informatics.predictors.predictor.Predictor` has been trained, a one-off prediction may be made against it by using the :func:`~citrine.informatics.predictors.predictor.Predictor.predict` method.
+
+This method accepts a :class:`~citrine.informatics.predictors.single_predict_request.SinglePredictRequest`, which is akin to a :ref:`DesignCandidate <design_candidate_anchor>` that you can define and modify and is not persisted in the Citrine Platform. When building a :class:`~citrine.informatics.predictors.single_predict_request.SinglePredictRequest` note that only the material properties required to make a prediction (the "input" properties") are required. Indeed, when making a prediction on a predictor using the :func:`~citrine.informatics.predictors.predictor.Predictor.predict` method, the system will automatically filter out any provided material properties that are not inputs to the predictor. The output of a call to ``predict()`` is a :class:`~~citrine.informatics.predictors.single_prediction.SinglePrediction`, which is essentially the :class:`~citrine.informatics.predictors.single_predict_request.SinglePredictRequest` with all of the predicted properties of the material filled in with the predicted values.
+
+Note that a ``random_seed`` may be provided to the :class:`~citrine.informatics.predictors.single_predict_request.SinglePredictRequest`. Providing a consistent ``random_seed`` across requests with the same inputs guantees consistent predictions.
+
+The following is a simple example of several predictions based on a function that builds a list of prediction requests. This example retrieves 3 candidates from a prior design execution, updates them slightly, and makes new predictions with the updated inputs. Note that while this example uses existing an :ref:`DesignCandidate <design_candidate_anchor>` as a convenience to build the update prediction requests, there is no requirement that a prediction request be related to an existing :ref:`DesignCandidate <design_candidate_anchor>` -- rather any arbitrary request can be made as long as the inputs satisfy the requirements of the predictor.
+
+.. code:: python
+
+   import os
+   from citrine import Citrine
+   from citrine.informatics.predictors.single_predict_request import SinglePredictRequest
+   from citrine.informatics.predictors.single_prediction import SinglePrediction
+
+   # assuming your Citrine deployment is https://matsci.citrine-platform.com
+   CITRINE_HOST="matsci.citrine-platform.com"
+   citrine_client = Citrine(host=CITRINE_HOST, api_key=os.environ.get("CITRINE_API_KEY"))
+
+   PROJECT_ID="<uuid>"
+   my_project = citrine_client.projects.get(PROJECT_ID)
+
+   # arbitrary example of building a list of requests
+   def build_requests() -> list[SinglePredictRequest]:
+     WORKFLOW_ID="<uuid>"
+     EXECUTION_ID="<uuid>"
+
+     my_workflow = my_project.design_workflows.get(WORKFLOW_ID)
+     my_execution = my_workflow.design_executions.get(EXECUTION_ID)
```

Comment:
This is a bit nit-picky, but in most of our code examples we don't put in the part about setting up your client or `get`-ing all of your assets. Instead, we just use variable names like `my_design_execution` as if you had already designated them somewhere to be your asset on platform.
I think this helps make the individual feature stand out more, but also I could see folks trying to include `<>` in the UID string.